test/jdk/jdk/incubator/vector/AbstractVectorConversionTest.java

```diff
@@ -48,7 +48,7 @@ public void getRunTime(ITestResult tr) {
         System.out.println(tr.getName() + " took " + time + " ms");
     }
 
-    static final int INVOC_COUNT = Integer.getInteger("jdk.incubator.vector.test.loop-iterations", 1000);
+    static final int INVOC_COUNT = Integer.getInteger("jdk.incubator.vector.test.loop-iterations", 100);
```

Comment:
Why did you change this?

Reply:
Following reasons:
The INVOC_COUNT is set as 100 for all other tests.
The conversion tests take longer time to complete than other tests.
For the conversion tests it looks like the intrinsics don't trigger sometimes and tests take longer to execute and timeout. This was one of the reasons for backout last time.

Reply:
ok